Strange test failure on JDK24 ...

```
org.eclipse.jetty.server.CustomRequestLogTest.testLogRequestTimeCustomFormats([2] %{EEE MMM dd HH:mm:ss zzz yyyy|EST}t)

java.lang.AssertionError: 
Expected: a value less than or equal to <2L>
     but: <3600L> was greater than <2L>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
	at org.eclipse.jetty.server@12.0.18-SNAPSHOT/org.eclipse.jetty.server.CustomRequestLogTest.testLogRequestTimeCustomFormats(CustomRequestLogTest.java:625)
```

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -28,13 +28,13 @@ pipeline {
           }
         }
 
-        stage("Build / Test - JDK23") {
+        stage("Build / Test - JDK24") {
           agent { node { label 'linux' } }
           steps {
             timeout( time: 180, unit: 'MINUTES' ) {
               checkout scm
-              mavenBuild( "jdk23", "clean install -Dspotbugs.skip=true -Djacoco.skip=true", "maven3")
-              recordIssues id: "jdk23", name: "Static Analysis jdk23", aggregatingResults: true, enabledForFailure: true, tools: [mavenConsole(), java(), checkStyle(), javaDoc()]
+              mavenBuild( "jdk24", "clean install -Dspotbugs.skip=true -Djacoco.skip=true", "maven3")
+              recordIssues id: "jdk24", name: "Static Analysis jdk24", aggregatingResults: true, enabledForFailure: true, tools: [mavenConsole(), java(), checkStyle(), javaDoc()]
             }
           }
         }

--- a/jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/CustomRequestLogTest.java
+++ b/jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/CustomRequestLogTest.java
@@ -594,7 +594,7 @@ public class CustomRequestLogTest
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {"%{EEE MMM dd HH:mm:ss zzz yyyy}t", "%{EEE MMM dd HH:mm:ss zzz yyyy|EST}t", "%{EEE MMM dd HH:mm:ss zzz yyyy|EST|ja}t"})
+    @ValueSource(strings = {"%{EEE MMM dd HH:mm:ss zzz yyyy}t", "%{EEE MMM dd HH:mm:ss zzz yyyy|America/New_York}t", "%{EEE MMM dd HH:mm:ss zzz yyyy|America/New_York|ja}t"})
     public void testLogRequestTimeCustomFormats(String format) throws Exception
     {
         AtomicLong requestTimeRef = new AtomicLong();

--- a/pom.xml
+++ b/pom.xml
@@ -263,7 +263,7 @@
     <maven.exec.plugin.version>3.5.0</maven.exec.plugin.version>
     <maven.gpg.plugin.version>3.2.7</maven.gpg.plugin.version>
     <maven.install.plugin.version>3.1.3</maven.install.plugin.version>
-    <maven.invoker.plugin.version>3.8.1</maven.invoker.plugin.version>
+    <maven.invoker.plugin.version>3.9.0</maven.invoker.plugin.version>
     <maven.jar.plugin.version>3.4.2</maven.jar.plugin.version>
     <maven.javadoc.plugin.version>3.11.2</maven.javadoc.plugin.version>
     <maven.plugin-tools.version>3.15.1</maven.plugin-tools.version>
@@ -1457,23 +1457,23 @@
               <localRepo>${localRepoPath}</localRepo>
             </filterProperties>
           </configuration>
-          <dependencies>
-            <dependency>
-              <groupId>org.apache.groovy</groupId>
-              <artifactId>groovy</artifactId>
-              <version>${groovy.version}</version>
-            </dependency>
-            <dependency>
-              <groupId>org.apache.groovy</groupId>
-              <artifactId>groovy-json</artifactId>
-              <version>${groovy.version}</version>
-            </dependency>
-            <dependency>
-              <groupId>org.apache.groovy</groupId>
-              <artifactId>groovy-xml</artifactId>
-              <version>${groovy.version}</version>
-            </dependency>
-          </dependencies>
+          <!--          <dependencies>-->
+          <!--            <dependency>-->
+          <!--              <groupId>org.apache.groovy</groupId>-->
+          <!--              <artifactId>groovy</artifactId>-->
+          <!--              <version>${groovy.version}</version>-->
+          <!--            </dependency>-->
+          <!--            <dependency>-->
+          <!--              <groupId>org.apache.groovy</groupId>-->
+          <!--              <artifactId>groovy-json</artifactId>-->
+          <!--              <version>${groovy.version}</version>-->
+          <!--            </dependency>-->
+          <!--            <dependency>-->
+          <!--              <groupId>org.apache.groovy</groupId>-->
+          <!--              <artifactId>groovy-xml</artifactId>-->
+          <!--              <version>${groovy.version}</version>-->
+          <!--            </dependency>-->
+          <!--          </dependencies>-->
         </plugin>
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>

